courses/free-courses-hi.md

```diff
@@ -402,6 +403,11 @@
 * [Web Development Course](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n) - Apna College
 
 
+### Express.js
+
+* [Express.js - Learn What Matters](https://youtu.be/pKJ4GGyDgJo?si=YUJf86FH54wofl99) - Sheryians Coding School
```

Comment:
we don't accept shortened links like this one. single videos are only acceptable if there is no playlist and the lengthe is longer than one hour, as this one is.
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. For this one, you can just add it to the javascript category